### DISCUSSION TITLE
Volume fraction of texture

### DISCUSSION THREAD

**Discussion by Dev049** [mtex-toolbox/mtex#2152] [Ask Anything]
I am new on Matlab. Just want to calculate volume fractio of textures using Mtex. I have raw EBSD data with me.

**Comment by kilir** [comment]
Hi,

a good point to start is probably the documenation (https://mtex-toolbox.github.io/Documentation), probably have a look how to compute an odf and then have a look at the function `SO3Fun/volume`.

Cheers,

Rüdiger
